src/components/Profile/ProfileMenu.js

```diff
@@ -18,6 +19,9 @@ export class ProfileMenu extends Component {
     } else if (this.state.activeItem === "following") {
       displayItem = <FollowersFollowing profile={profile} follow_list="following" />
     }
+    else if (this.state.activeItem === "settings") {
```

Comment:
Consider changing the title from settings to subscribe.
The tab is also displayed when viewing other users' profiles. I think it should only show on the current user's profile or you can consider changing it to be at the navigation bar instead